script/configs/allowed_unpinned_deps.yaml

```diff
@@ -60,6 +60,7 @@
 - wasm
 - yaml
 # Google-owned packages
+- _discoveryapis_commons
```

Comment:
@keyonghan I had to add `_discoveryapis_commons` to the allowlist, let me know if there are any concerns with this

Reply:
LGTM.